raster-foundry/raster-foundry
# Improve supported CRS for WMS and WCS Services

## Overview

This PR:
- uses image's original `CRS` when transforming to `MosaicRasterSource `
- appends image's `CRS` to the list of `supportedProjections` to `WmsParentLayerMeta`
- fixes a bug in specifying band numbers in single band project layer for OGC styles

### Checklist


### Notes

It keeps the assumption that the images are from the same datasource and share the same original CRS.

## Testing Instructions
- `backsplash-server/assembly`, `api/assembly`, `mg apply` (if you have not done so in a while)
- Spin up server and frontend
- Add an image to a project layer, e.g. an image with polar projection system described in the issue
- Make this project public
- `gdalinfo <imageName.extension>` and check its CRS
- `GET` to `http://localhost:8081/wms/<projectID>?SERVICE=WMS&REQUEST=GetCapabilities` -- check that the `<CRS>` tag contains the image's original CRS epsg code in addition to epsg codes of WGS84 and WebMercator
- `GET` to `http://localhost:8081/wcs/<projectID>?SERVICE=WCS&REQUEST=GetCapabilities` -- check that the `<SupportedCRS>` tag contains the image's original CRS epsg code


Closes https://github.com/raster-foundry/raster-foundry/issues/4831
